### PR TITLE
Added missing #includes on OpenBSD

### PR DESCRIPTION
**Pull request by Ronaldr1985**
Resolves issues with compiling Odin on OpenBSD, sys/resource.h and sys/wait.h are required for WIFEXITED and WEXITSTATUS.

### CODE REVIEW

File: src/common.cpp [hunk 1]
```diff
@@ -25,6 +25,11 @@
 #include <psapi.h>
 #endif
 
+#if defined(GB_SYSTEM_OPENBSD)
+#include <sys/resource.h>
+#include <sys/wait.h>
```
Comment (semarie) [inline src/common.cpp:30]: only `sys/wait.h` should be required for `WIFEXITED` and `WEXITSTATUS` macro.

also, I would put the missing includes inside `src/gb/gb.h` under the `#if defined(GB_SYSTEM_OPENBSD)`

File: build_odin.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (semarie) [inline build_odin.sh:1]: I agree that it is the more portable way to invoke `bash`

**Review by semarie — CHANGES_REQUESTED** [review]

**Comment by semarie** [comment]
@Ronaldr1985 ping ?

**Comment by Ronaldr1985** [comment]
Apologies for the large delay, I was busy and then I got sick unfortunately.

However, I have moved the include to gb/gb.h and removed it from common.cpp.

File: build_odin.sh (hunk 2 above)
Comment (Ronaldr1985) [reply build_odin.sh:1]: Yes, this means that the script can still run on OpenBSD without requiring changes to the SHABANG.

**Pull request merged**